webservice/utils/get-data-from-form-section.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var assign = require('es5-ext/object/assign');
+
+module.exports = function getDataFromSection(dataForm, result, typeMapper) {
```

Comment:
OK, so we have `getDataFromSection` in the module which is called `get-data-from-form-section`... so which is it `FromSection` or `FromFormSection`?